### PR TITLE
Deprecate named arg to locally

### PR DESCRIPTION
**Pull request by som-snytt**
It is an unlikely snare into which I was led
by a perverse sense of adventure.

### CODE REVIEW

File: src/library/scala/Predef.scala [hunk 1]
```diff
@@ -249,7 +249,7 @@ object Predef extends LowPriorityImplicits {
    *           }}}
    *  @group utilities
    */
-  @inline def locally[T](x: T): T = x
+  @inline def locally[T](@deprecatedName("x") x: T): T = x
```
Comment (hrhino) [inline src/library/scala/Predef.scala:252]: I almost expected `def locally[T](@deprecatedName x: T)` to work.

**Review by hrhino — APPROVED** [review]
"Whereof one cannot speak, thereof one must be silent."

File: src/library/scala/Predef.scala (hunk 1 above)
Comment (som-snytt) [reply src/library/scala/Predef.scala:252]: By any other name, it would be a code smell.

**Comment by lrytz** [comment]
Nice one :)

**Pull request merged**